src/thread/setns.rs

```diff
@@ -0,0 +1,122 @@
+#![allow(unsafe_code)]
+
+use core::convert::TryFrom;
+
+use bitflags::bitflags;
+
+use crate::backend::c::c_int;
+use crate::backend::thread::syscalls;
+use crate::fd::BorrowedFd;
+use crate::io;
+
+/// New time namespace.
+const CLONE_NEWTIME: u32 = 0x00_00_00_80;
```

Comment:
Can these constants use `linux_raw_sys::general::CLONE_NEWTIME` and similar instead of hard-coding the constants here?